overlord/snapstate/snapstate.go

```diff
@@ -4148,9 +4148,9 @@ func GadgetConnections(st *state.State, deviceCtx DeviceContext) ([]gadget.Conne
 // for all current snaps in the system state.
 //
 // A downloaded file is only kept if any of the following are true:
-//    1. The revision is in SnapState.Sequence
-//    2. The revision is in saved in refresh-candidates
-//    3. The revision is pointed to by a download task in an ongoing change
+//  1. The revision is in SnapState.Sequence
+//  2. The revision is in saved in refresh-candidates
+//  3. The revision is pointed to by a download task in an ongoing change
```

Comment:
where does this change come from?

Reply:
It was changed in master by commit 018cdb4cc401367544d6cf67699a4ff74c1ed615, and it seems my linter automatically changed the indentation again